g2gen/Cargo.toml

```diff
@@ -12,7 +12,7 @@ A macro to create types that implement fast finite field arithmetic.
 """
 categories = [ "no-std", "algorithms" ]
 keywords = [ "finite-field", "galois", "macro", "newtype"]
-rust-version = "1.61"
+rust-version = "1.70"
```

Comment:
```
cargo +1.61.0 check
```
Runs just fine. Issue seems to be running `cargo +1.61.0 test`, which I'd say can be ignored. I guess that means the CI setup should be updated so it only runs `check` in the MSRV case.

Reply:
In that case, I think we need to pin dependencies to an older version, cause I get

> error: package `textwrap v0.16.2` cannot be built because it requires rustc 1.70 or newer, while the currently active rustc version is 1.61.0

when trying to run the tests with `cargo +1.61.0 test`

Reply:
Yes. the dev-dependecies are no longer compatible with 1.61.0. So you cannot run `cargo +1.61.0 test`. But you can still run `cargo +1.61.0 check`. So `g2p` will run just fine with the older rust version, it's just that we can't test it with the current tests :/